scripts/build-cdn-indexes.ts

```diff
@@ -0,0 +1,74 @@
+import path from 'path';
+import themes from '../themes.json';
+import fs from 'node:fs';
+import c from 'chalk';
+
+const getSystems = () => {
+  const result: string[][] = [];
+  Object.entries(themes).forEach((theme) => {
+    const config = theme[1];
+    if ('incomplete' in config) return;
+
+    const shortname =
```

Comment:
Looks like the only place this is used is to get the theme file name, which should actually match the key in this object. The `config.packageName.split('-')[1]` won't work for `cmsgov-theme.css` as far as I can tell. Instead of having any logic here at all, could you change the parameters part to something like

```
Object.entries(themes).forEach(([themeName, themeInfo]) => {
...
result.push([themeInfo.packageName, version, themeName]);
```

Reply:
Hmm I was thinking it would be called `ds-cms-gov`, ok, yeah that's easier in that case.